### ISSUE TITLE
Bar Chart x-axis is disappearing in IDS 4.85.0

### ISSUE THREAD

**Issue by nakula1** [infor-design/enterprise#7797]
**Describe the bug**
Bar chart x-axis values are disappearing after we enable the preview mode in Infor OS Portal.

**To Reproduce**

1. Add a widget with chart https://design.infor.com/code/ids-enterprise/latest/bar
2. Open the Infor OS portal in preview mode. adding **?ospClientPreview=true**to the url
3. You'll see the x-axis values of chart will disappear.

**Expected behavior**
x-axis values should not disappear.

**Version**

- IDS version changes from 4.80.2 to 4.85.0 when we enable preview mode. We have issue in IDS 4.85.0.
- On preview mode, Infor OS version is 2023.09.00 (PREVIEW, 20230727-131728, 3063, 8388)

**Screenshots**
<img width="278" alt="image" src="https://github.com/infor-design/enterprise/assets/141989104/466d7462-cf8e-4a8d-9dd4-3b7166f4549a">

**Platform**
 - Infor Application/Team Name: Infor OS Workspace, Infor AIS

**Comment by tmcconechy** [comment]
@nakula1 we dont have a way to run the widget toolkit as we work on a lower level. Can you make a reduced test case?
This means copy the relevent code to reproduce the issue (and only that code). To a runnable template like this one https://stackblitz.com/edit/ids-quick-start-1661

If i was to guess. Try and check the parent element the chart is in and see if you have the correct height and width in css that fits the view port. Or try and change the height and width.

Also check the latest version 4.86 as it may possibly be fixed as we had issues in this area

**Comment by tmcconechy** [comment]
@nakula1 your example was helpful the reason its not showing is that you did not provide data for the legend name.

```
  public barData = [
    {
      data: [
        {
          name: 'Category A',
          value: 373,
          url: 'test',
        },
        {
          name: 'Category B',
          value: 372,
        },
        {
          name: 'Category C',
          value: 236.35,
        },
      ],
      name: 'Categories',  <- add this or a name for it
    },
  ];
```

**Comment by nakula1** [comment]
@tmcconechy , I don't think so. May the developer who provided this sample doesn't cover all. If you refer to the screen shot below. We are actually getting the x-axis values and when we enable the OS preview mode, it is disappearing. Not sure if this is IDS issue or the widget SDK/OS portal side issue.
<img width="220" alt="image" src="https://github.com/infor-design/enterprise/assets/141989104/e39ba16a-18c0-46a0-9249-947c5ed9ddaa">

Let me confirm again with our development team. I can update on Monday.

**Comment by nakula1** [comment]
We have checked the height and it was not related to height. We have same chart in other existing widgets as well and the issue is even in those widgets.

**Comment by tmcconechy** [comment]
ok then please update that stackblitz so it shows the issue. thats the only way we will be able to look at it.

**Comment by tmcconechy** [comment]
If possible can you take a look @ericangeles

**Comment by ericangeles** [comment]
@nakula1 has the stackblitz been updated?

**Comment by nakula1** [comment]
HI @ericangeles, I think @tmcconechy was able to reproduce it. Our team had a conversation in Teams channel.
<img width="409" alt="image" src="https://github.com/infor-design/enterprise/assets/141989104/d0bd7af8-133f-428c-b83d-766806de58ee">

**Comment by nakula1** [comment]
Is this expected to be resolved in IDS 4.87.x version? and any timeline on when this is expected to be in OS Portal?
